src/components/TextInput/TextInput.tsx

```diff
@@ -192,6 +199,24 @@ export const TextInput = ({
                 data-test-id="text-input"
                 {...spellcheckProp}
             />
+            {extraAction && (
+                <button
+                    className={merge([
+                        'tw-flex tw-items-center tw-justify-center tw-transition-colors tw-rounded',
+                        disabled ? 'tw-cursor-default tw-text-black-40' : 'tw-text-black-60  hover:tw-text-black-100',
+                        extraActionButtonIsFocusVisible && FOCUS_STYLE,
+                    ])}
+                    onClick={() => extraAction.onClick()}
+                    data-test-id="extra-action-icon"
+                    title={extraAction.title}
```

Comment:
![screenshot_2023-04-17_at_15 46 35](https://user-images.githubusercontent.com/6846950/232741687-0c207047-c19b-42f3-a18c-87a8dac8b56e.png)

Should probably be a `Tooltip`, if so we should align it for `clearable` and `copyable` buttons